### PR TITLE
Add test for issue #1899

### PR DESCRIPTION
**Pull request by krwq**
Testing issue reported in the issue #1899. Does not repro.

### CODE REVIEW

File: src/System.Xml.XmlDocument/tests/example.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      George W. Bush
+    </Student>
+    <Student id="5" score="99" enrollment="1/20/2009" comment=
+             "Ridiculously low interest rate, high house price. $ worthless">
+      Barack H. Obama
+    </Student>
+    <Student id="6" score="35" enrollment="4/30/2011"
+             comment="Never qualified to be the president.">Song Li</Student>
+  </Studentlist>
+</StudentsInformation>
```
Comment (stephentoub) [inline src/System.Xml.XmlDocument/tests/example.xml]: I realize it's just meant to be arbitrary data, but let's avoid anything close to political commentary.

File: src/System.Xml.XmlDocument/tests/RandomTests.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿using System.IO;
+using Xunit;
+
+namespace System.Xml.Tests
+{
+    public class RandomTests
```
Comment (stephentoub) [inline src/System.Xml.XmlDocument/tests/RandomTests.cs]: Is there some name more descriptive we could use? Or an existing file this could go in?

File: src/System.Xml.XmlDocument/src/project.lock.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```
Comment (stephentoub) [inline src/System.Xml.XmlDocument/src/project.lock.json]: Was this change on purpose?

File: src/System.Xml.XmlDocument/tests/project.lock.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```
Comment (stephentoub) [inline src/System.Xml.XmlDocument/tests/project.lock.json]: Was this change on purpose?

**Comment by stephentoub** [comment]
Other than my question on the locked in the project.lock.json files, LGTM.

File: src/System.Xml.XmlDocument/src/project.lock.json (hunk 3 above)
Comment (krwq) [inline src/System.Xml.XmlDocument/src/project.lock.json]: I did not manually change it

File: src/System.Xml.XmlDocument/tests/project.lock.json (hunk 4 above)
Comment (krwq) [inline src/System.Xml.XmlDocument/tests/project.lock.json]: Not intended, no idea what this means, just got auto changed when I built so I decided to add this to review

File: src/System.Xml.XmlDocument/src/project.lock.json (hunk 3 above)
Comment (stephentoub) [inline src/System.Xml.XmlDocument/src/project.lock.json]: I understand.  But it got updated somehow, and I believe our policy is still to have these checked in as locked: true.  @weshaggard can clarify.

**Comment by krwq** [comment]
Removed json files changes

**Comment by krwq** [comment]
Build failure not related to this change

**Comment by stephentoub** [comment]
> Build failure not related to this change

What was it?  Is there an issue logged for it?
